src/Mobs/Monster.cpp

```diff
@@ -384,6 +385,24 @@ void cMonster::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 
 	BroadcastMovementUpdate();
 
+	// Ambient mob sounds
+	if (!m_SoundAmbient.empty())
+	{
+		--m_AmbientSoundTimer;
+
+		if (m_AmbientSoundTimer <= 0)
+		{
+			auto & Random = GetRandomProvider();
+			auto ShouldPlaySound = Random.RandBool();
+			if (ShouldPlaySound)
+			{
+				auto SoundPitchMultiplier = 1.0f + (Random.RandReal(1.0f) - Random.RandReal(1.0f)) * 0.2f;
+				m_World->BroadcastSoundEffect(m_SoundAmbient, GetPosition(), 1.0f, SoundPitchMultiplier * 1.0f);
```

Comment:
A multiplication by one shouldn't do anything to SoundPitchMultiplier?

Reply:
Ah right, leftover from my explosion changes. Will fix.